postgresql/initdb.d/04_grants.sql

```diff
@@ -158,10 +158,29 @@ GRANT SELECT ON local_ega_ebi.file_dataset TO download;
 
 --------------------------------------------------------------------------------
 
+CREATE ROLE api;
+
+GRANT USAGE ON SCHEMA sda TO api;
+GRANT SELECT ON sda.files TO api;
+GRANT SELECT ON sda.file_dataset TO api;
+GRANT SELECT ON sda.checksums TO api;
+GRANT SELECT ON sda.file_event_log TO api;
```

Comment:
API also needs to be able to read the datasets and the statuses of the datasets see #1080